R/estimateDiversity.R

```diff
@@ -189,12 +189,17 @@
 #'
 #' # Calculate diversities
 #' tse <- estimateDiversity(tse, index = index)
-#'
+#' 
+#' # Verify the columns in colData
+#' col_names <- colnames(colData(tse))
+#' valid_indices <- index[index %in% col_names]
+#' 
 #' # The colData contains the indices with their code names by default
-#' colData(tse)[, index]
+#' colData(tse)[, valid_indices]
 #'
 #' # Removing indices
-#' colData(tse)[, index] <- NULL
+#' colData(tse)[, valid_indices] <- NULL
+#' colData(tse)[, name] <- NULL
```

Comment:
name was not yet used, so this line is extra (there cannot be index called name)

Reply:
removed ...was made during some testing